### ISSUE TITLE
I am confusion.

### ISSUE THREAD

**Issue by TakahashiEdenyako** [niacdoial/blemd#41]
Hi, I'm really sorry, but I don't know anything. I tried to follow your instructions, but I got so confused with everything. I'm sorry if that comes across as rude to your instructions. I'm trying to hack Pikmin 2 and change for example the Valley of Repose's model. I watched a video tutorial where I could extract the files through Luna boy, then I'd have an arc folder containing the model. I tried clicking on the model and it had a blender icon next to it. I clicked on it, it opened for a half a second. Then I watched the rest of the tutorial and it said to get a BMD Importer. I'm so confused and I don't really know how to use Git hub. I clicked download, then I clicked download zip. then I extracted its zip into my computer's document folder, just documents nothing else other than that. (I did put it in an organized folder specifically for BMD importer) Now that I have it in there, I am confusion. I don't know what anything means or how to use BMD or what to do with it and blender. If you could, carefully show me every step by step with either a video or images, that'd be great. I'm pretty sure I haven't made my self clear on the problem: there is a folder in my files called "blend-master", then when I open that file, other files such as "automated-converter", "Image importing subprocess", ".gitingore." And a bunch of other files that have a white paper icon next to their name. I would like to understand how I can use these files or whatever I have to do to them, in order to allow me to import the Valley of Reposes model into Blender. I'm sorry for not being really smart with this. I'm new to hacking and stuff and it's really confusing to me and what to send things do, what to extract, where to put where, so please send me a guide with images or a video of everything to do with the BMD importer files. I tend to struggle with directions especially when it's not visual. -Some dumb 16-year old. 
Post Script: No, the video I had been watching didn't feature how to download anything, it just showed them importing the Perplexing Pool into blender, but it said that the file they used was with BMD Importer, then I tried to look it up, found this and now I don't know anything.

**Comment by ajd-262** [comment]
For blemd I think you need Blender 2.79 or older. How I got it working for Blender 2.78 is to take the blemd-master folder from inside the zip and copy and paste it to Blender's addons_contrib folder. For my Blender install that is C:\3D\blender-2.78c-windows64\2.78\scripts\addons_contrib.

After that start Blender than goto File> User Preferences. Click on the Add-on tab. Next click on the Testing tab. You should now see "Import-Export: Import gc/wii bmd fFormat (.bmd, .bdl)" Check mark it and you should now have  Nintendo BMD/BDL listed in Import

**Comment by TakahashiEdenyako** [comment]
I think I can understand most of what you’re saying. So, copy and paste “blemd-master” into Blender’s addon contrib folder. But what and where is the addon_contrib folder? Should it be in files, or when I literally go to my pc’s desktop, click on Blender and open it up? I get what you’re saying, just where exactly did you find Blender_addon_contrib folder? All I know that I have on my computer is that the Blender app is in the folders in the desktop tab. But I don’t see any files for it, do you know where that info is. I sound like a noob...

**Comment by ajd-262** [comment]
You need to go to the location that Blender is installed. There is no shortcut for this with in Blender to do this.  Once you find the main Blender folder it should look like the attach image. If you using Blender 2.79 the folder will be named 2.79. Inside of the folder is a scripts folder. Inside of that is the addons_contrib.

![Blender](https://user-images.githubusercontent.com/46447785/120092530-6bd44e00-c0c8-11eb-8cfd-32295f39e3a1.png)

**Comment by niacdoial** [comment]
er… thanks for answering in my stead, ajd-262!
The only think I need to add is that, one way to find where blender is installed is this:
- right click on the shortcut that allows you to open blender
- choose "open link location" (or something similar)
and you should be in the folder like the one from the image above.

Also, once you put the `blemd-master` folder into `addons_contrib`, you need to open blender, to to File->Preferences,
then Addons in the window that opens up,  then click on the "testing" button. At that point, the add-on should show up somewhere in the list of addons. You will just have to enable it, click something like "save user preferences", and you will be good to go.

**Comment by TakahashiEdenyako** [comment]
Thank you very much. I appreciate it a lot.^_^

**Comment by niacdoial** [comment]
short version: I assume you have a recent version of windows, so take either blender-2.78c-windows64.msi (if you are admin of your machine) or blender-2.78c-windows64.zip (if you aren't an admin, or if you want to control where blender gets installed).

Long version: you need to look at the name of the files, but know you only need one of them.
- the difference between "2.78", "2.78a", "2.78b" and "2.78c" is that, with each letter change, the blender devs fixed more bugs.
- then, you have the choice between "windows", "OSX" (mac), and "linux". You need to choose a version of blender that works with whatever OS you have on your PC.
- for the "32" or "64" at the end of "windows", it's just if you want a 32-bit installation or a 64-bit intallation of blender. Most recent-ish computers (built after 2010) can use 64-bit programs, but 32-bit programs will work everywhere, (but sometimes they can't open very large projects)
- finally, ".msi" means it's a windows installer, and ".zip" means it's zip archive that contains a pre-installed blender. You just need to move it outside of the archive, and it will work.

**Comment by TakahashiEdenyako** [comment]
I have a surface pro 6.

**Comment by niacdoial** [comment]
…
OK, so.
"surface pro 6" is the name of the PC 'as a machine', which means it is the name of the "parts that you can touch" (screen+keyboard+touchpad+CPU+ram+etc…).
What I'm interested in is the Operating System (aka OS), aka the 'program' that shows you the desktop, allows you to move the windows around, that controls what show up on the screen and how, lets you log in and log out, and many more things.
I'm fairly sure that the OS you use is a recent version of windows.

**Comment by TakahashiEdenyako** [comment]
Yes, I have  a recent version of windows (I’m not really good with computers...obviously) also, what is an admin and how do I use that?

**Comment by niacdoial** [comment]
OK er…
An admin is not something you use, it something you are, or aren't.

imagine that your PC is like a hotel.
if there are multiple people that sleep in it, they will have different rooms and such. Each person who has a room is like an user for your  PC.
Most people in there (most users) can't do a lot of things outside of their room. Like, they can't change the elevators, build a swimming pool, etc…
But there is at least someone who runs the hotel, and who *can* do that.
In your PC, that person is an "admin user", and they can install or remove programs, and change some obscure settings. 

Basically, an admin is a person that controls a lot of things in the PC, where other people can't. (But of course, with great power…)

**Comment by TakahashiEdenyako** [comment]
Everything seems good. I clicked on the Nintendo thing in the import section, then it let me go to my documents and click on where the Valley of Repose’s model is, but it’s not loading and it’s staying on the same screen where its file is and it’s highlighted in Orange.

**Comment by niacdoial** [comment]
…double click, or click the "Import"/"Open" button?

**Comment by niacdoial** [comment]
also, it might take more than a minute, if the model is large, or if there are many animations to load

**Comment by TakahashiEdenyako** [comment]
When I double click it, it says blender not responding on the top left

**Comment by niacdoial** [comment]
wait hold up.
er… this is not the first time

**Comment by TakahashiEdenyako** [comment]
I’ve tried it several times

**Comment by niacdoial** [comment]
er… does it work if you use the "do not import animations" option while importing the model?
(like in #38?)

**Comment by TakahashiEdenyako** [comment]
I’m pretty sure I did it correctly. I extracted the model to Lunaboy and I got model.bmd, then I did your steps in blender, went to where it says Nintendo, and clicked its model

**Comment by TakahashiEdenyako** [comment]
How do I not import animations and what is #3 8

**Comment by niacdoial** [comment]
just before you click the bmd file in blender, there should be a bunch of options in a side panel of the window.
one of them (a drop-down menu) talks about how animations are imported, and one of the options is to not import them.

**Comment by niacdoial** [comment]
also, for the number 38, that's a link towards another discussion where someone had the same problem as you.

**Comment by TakahashiEdenyako** [comment]
![Screenshot (2)](https://user-images.githubusercontent.com/85026028/120105560-707c1f00-c127-11eb-840a-17ed9c4628ec.png)

Sorry for the small screen, I don't know how to fix it, but don't worry about that. Where is the side panel you're talking about? I don't see it.

**Comment by niacdoial** [comment]
bottom-left, where you already have your cursor.
the rectangle with a scrollbar, with "Import BMD" at the top is the sidepanel I was talking about.
click on "single action", change it into "do not import animations."

**Comment by TakahashiEdenyako** [comment]
I tried that but it still says Blender (Not Responding)

**Comment by niacdoial** [comment]
you need to do that before clicking on the file
close blender and re-open it if it's not responding.

**Comment by TakahashiEdenyako** [comment]
I did that too. Also, how many times should I click on the model.bmd? Or do I press enter?

**Comment by niacdoial** [comment]
I…
1: close and reopen blender if not responding.
2: click on the "import -> nintendo BMD" option
3: go to the right folder
4: change the options *now*. Not afterwards
5: double click on the file.
does it work now?

**Comment by TakahashiEdenyako** [comment]
Okay, I did that. When I got to opening the file, I clicked it once and it turned orange then I clicked it again and it’s still orange. But I know if I were to click it more, it would say not responding

**Comment by niacdoial** [comment]
OK, first click selects the file, second click starts the import  (which makes blender unresponsive), and windows *realises* it became unresponsive at the third click.

**Comment by niacdoial** [comment]
maybe you would have more luck if you try and get BleMD from [here](https://github.com/niacdoial/blemd/tree/0d618dd6743b7fc3edefe47777fee855f6335531)?
(it's a sightly older version)
at this point I'm just throwing things to the wall, and seeing what sticks

**Comment by TakahashiEdenyako** [comment]
Oh good! Do you know how to move the camera around?

**Comment by niacdoial** [comment]
that… would be where others would probably help you better than me.
if you want to learn how to use blender, I can suggest https://www.youtube.com/watch?v=TPrnSACiTJ4. (skip to 1:55 to skip the intro)

also, my next message will be me talking to myself, ignore it

**Comment by niacdoial** [comment]
lesson 1 from this: #38 is indeed related to #33.

**Comment by TakahashiEdenyako** [comment]
Thanks, I really appreciate all your help and patience. I can’t thank you enough. You’ve been really helpful.

**Comment by niacdoial** [comment]
you're welcome!
